src/diffusers/models/controlnets/controlnet_sd3.py

```diff
@@ -56,6 +56,10 @@ def __init__(
         pooled_projection_dim: int = 2048,
         out_channels: int = 16,
         pos_embed_max_size: int = 96,
+        dual_attention_layers: Tuple[
```

Comment:
we already have this change https://github.com/huggingface/diffusers/blob/963ffca43419a8dffa682d9e03c2299b76feeced/src/diffusers/models/controlnets/controlnet_sd3.py#L60

can you rebase?

Reply:
@yiyixuxu done.